### ISSUE TITLE
SSL Certificates for securing website

### ISSUE THREAD

**Issue by Miteshdv** [aws-samples/amazon-cloudfront-secure-static-site#26]
Do we have the SSL Certificates integrated by default so that all our traffic is navigated from www.example.com to https://example.com by default ?

**Comment by thomasturrell** [comment]
Yes an AWS certificate is automatically created and the CloudFront distribution is configured to use it and redirect traffics from http to https.

What it won’t do is redirect traffic from the sub domain (www in your question) to the apex. You will have to manually add an alias to CloudFront and add an entry to Route 53 yourself if you want to do that.

**Comment by allan2** [comment]
Just came across this awesome template.

Could it be updated to support the naked domain?
i.e. include the naked domain on the certificate, as a CloudFront alias, and as a CNAME entry.

It would be nice to have https://www.domain.com and https://mydomain.com both working out of the box.

**Comment by ConnorKirk** [comment]
Thanks @thomasturrell for the answer.
